jupytercalpoly/Jupyter-Bifrost
# Data Tab State Machine

## Changes
- Reimplimented Data Screen menus to be mutually exclusive. Fixed bug where encodings and fields menu would show at the same time
- Fixed some errors due to rebasing where extraneous variables blocked `yarn build`
- Updated the `jupyter-bifrost-tracing` branch in the repo to reflect the new main branch.

## Testing
- Verify that the data tab menus are mutually exclusive: ex click on field and then encoding and make sure that only one menu shows up.
- Verify that clicking on the same encoding/field etc on the same pill toggles the menu.
